package.json

```diff
@@ -44,6 +44,7 @@
     "@testing-library/user-event": "^7.1.2",
     "@uniswap/widgets": "^2.47.3",
     "@unstoppabledomains/resolution": "8.5.0",
+    "@web3-name-sdk/core": "^0.1.18",
     "@web3-onboard/coinbase": "^2.2.5",
```

Comment:
@mishramonalisha76 the ticket description says resolved the domain name using space id package, but where is the space id package used?

Reply:
"@web3-name-sdk/core" this the space id package